src/CLI/CLI.php

```diff
@@ -536,7 +550,16 @@ public function run(?string $name = null, ...$args): void
 			return;
 		}
 
-		$command['command']($this);
+		try {
+			$command['command']($this);
+		} catch (Throwable $e) {
+			if ($this->climate->arguments->defined('debug') === true) {
+				throw $e;
+			}
+
+			$this->error($e->getMessage());
+			exit;
```

Comment:
Can we DRY this in a method like `handleError()`?

Reply:
Do you have an idea how we handle in `catch` block? 🙈 

Is this correct? @lukasbestle 
````php
try {
  $command['command']($this);
} catch (Throwable $e) {
  $this->handleException($e);
}

protected function handleException(Throwable $e): never
{
  if ($this->climate->arguments->defined('debug') === true) {
    throw $e;
  }

  $this->error($e->getMessage());
  exit;
}
````

Reply:
Hm, that's weird...

Why is it unhandled even though it is caught explicitly? TBH I don't know what the warning is trying to tell us here.

Reply:
OK. I figure out. IDE gives me warning because we throw exception in `handleException` method. So that's fine since we check debug mode. If debug mode enable, no problem throwing exception after all. I've pushed related commit https://github.com/getkirby/cli/pull/65/commits/c701b09b2182f1f3be55e160e95debc3053f5a0c